src/python/py/models/builder.py

```diff
@@ -2859,6 +2869,7 @@ def get_args():
                     is the prerequisite for the CUDA graph to be used correctly. It is not guaranteed that cuda graph be enabled as it depends on the model
                     and the graph structure.
                 use_8bits_moe = 1 : Use 8-bit quantization for MoE layers. Default is using 4-bit quantization.
+                hf_token = 1 : Use this to enable or disable the HF Authentication, can be True,False or the token itself.
```

Comment:
```suggestion
                hf_token = false/token: Use this to disable authentication with Hugging Face or provide a custom authentication token that differs from the one stored in your environment. Default behavior is to use the authentication token stored in the `HF_TOKEN` environment variable.
                    If you have already authenticated via `huggingface-cli login`, you do not need to use this flag because Hugging Face has already stored your authentication token in `HF_TOKEN` for you.
```